packages/calcite-components/src/components/tab-nav/tab-nav.tsx

```diff
@@ -55,19 +56,23 @@ export class TabNav {
   @Prop({ mutable: true }) selectedTitle: HTMLCalciteTabTitleElement = null;
 
   /**
+   * Specifies the size of the component inherited from the parent `calcite-tabs`, defaults to `m`.
+   *
    * @internal
    */
-  @Prop({ reflect: true, mutable: true }) scale: Scale = "m";
+  @Prop() scale: Scale = "m";
 
   /**
    * @internal
    */
   @Prop({ reflect: true, mutable: true }) layout: TabLayout = "inline";
 
   /**
-   * @internal
+   * Specifies the position of `calcite-tab-nav` and `calcite-tab-title` components in relation to, and is inherited from the parent `calcite-tabs`, defaults to `top`.
+   *
+   *  @internal
    */
-  @Prop({ reflect: true, mutable: true }) position: TabPosition = "bottom";
+  @Prop() position: TabPosition = "top";
```

Comment:
How come the default was flipped here?

Reply:
The usual. 
🎩 🪄 🐇

Reply:
No it's not. Must have been testing something and left it accidentally. I just commutes a  revert, if I haven't pushed it yet, I will when addressing the new feedback.